### ISSUE TITLE
Meross 

### ISSUE THREAD

**Issue by Marcoletto** [albertogeniola/meross-homeassistant#288]
**Describe the bug**
HA not accept the unit A & V

**Your environment**
HomeAssistant version: core-2021.11.5
Hardware environment: Rasp 3+

**Logs taken when the issue happened**
```
Logger: homeassistant.components.sensor.recorder
Source: components/sensor/recorder.py:269 
Integration: Sensore (documentation, issues) 
First occurred: 16:55:10 (2 occurrences) 
Last logged: 16:55:10

sensor.bagnetto_mss310_power_sensor_a_0 has unknown unit A
sensor.bagnetto_mss310_power_sensor_v_0 has unknown unit V
```

Altro: L'unità ('A') di questa entità non corrisponde a un'unità del.....

How can I solve?

**Comment by albertogeniola** [comment]
Hi,
This has been solved by latest version: v1.2.0rc0. Can you please upgrade and tell if that solves the issue?

**Comment by Marcoletto** [comment]
Yes, It's OK
Thank you for your work
